### DISCUSSION TITLE
ThaumicJEI / Thaumcraft API fixes

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#469] [Bugfix Suggestions]
- https://github.com/Buuz135/Thaumic-JEI/issues/34
  - https://github.com/Azanor/thaumcraft-api/issues/46
- https://github.com/Buuz135/Thaumic-JEI/issues/40

**Comment by ACGaming** [comment]
Handled by Thaumcraft Fix or ThaumicTweaker.
